server/src/main/java/org/opensearch/cluster/service/ClusterManagerThrottlingStats.java

```diff
@@ -39,4 +50,38 @@ public long getTotalThrottledTaskCount() {
     public void onThrottle(String type, int counts) {
         incrementThrottlingCount(type, counts);
     }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        out.writeInt(throttledTasksCount.size());
+        for (Map.Entry<String, CounterMetric> entry : throttledTasksCount.entrySet()) {
+            out.writeString(entry.getKey());
+            out.writeLong(entry.getValue().count());
+        }
+    }
+
+    public ClusterManagerThrottlingStats(StreamInput in) throws IOException {
+        int throttledTaskEntries = in.readInt();
+        throttledTasksCount = new ConcurrentHashMap<>();
+        for (int i = 0; i < throttledTaskEntries; i++) {
+            String taskType = in.readString();
+            long throttledTaskCount = in.readLong();
+            onThrottle(taskType, (int) throttledTaskCount);
+        }
+    }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params params) throws IOException {
+        builder.startObject("cluster_manager_throttling");
+        builder.startObject("cluster_manager_stats");
```

Comment:
just stats?

Reply:
Actually this stats are captured on cluster manager, so wanted to highlight that this are cluster manager stats, so that's why kept it as `cluster_manager_stats`. In future if we want add any stats from data nodes we can add new object called `data_node_stats`.

Reply:
Yes, that would be nested under parent object. Something similar to: 

```
"cluster_manager_throttling" : {
        "cluster_manager_stats" : {
         }, 
         "data_node_stats" : {
         }
     }
```

As of now we don't have any data node stats planned, but just wanted to keep this extensible in case if we want to add data node stats in it.

Reply:
Inside cluster_manager_throttling. i am not sure data_node_stats make sense. Also, it won't be extensible to maintain stats per node role type here.

Reply:
I was treating `cluster_manager_throttling` as feature name.

I agree, it will add confusion if we add `data_node_stats` in it. To keep it simple I will keep it as stats. 

```
"cluster_manager_throttling" : {
        "stats" : {
         }
     }
```